### PR TITLE
minor General Config GUI improvements

### PR DESCRIPTION
**Pull request by Simek**
- theme and language selectors wrapped into one GridBagSizer for better
appearance
- missing left padding added to checkbox on Advanced Pane

### CODE REVIEW

File: Source/Core/DolphinWX/Config/InterfaceConfigPane.cpp [hunk 1]
```diff
@@ -9,6 +9,7 @@
 #include <wx/choice.h>
 #include <wx/language.h>
 #include <wx/sizer.h>
+#include <wx/gbsizer.h>
```
Comment (lioncash) [inline Source/Core/DolphinWX/Config/InterfaceConfigPane.cpp]: Please sort this alphabetically,

Also consider putting up images of the differences before and after the change.

**Comment by aserna3** [comment]
If there are any noticeable visual changes to the UI, can you post screenshots?

**Comment by Simek** [comment]
Before and after screen

**Comment by aserna3** [comment]
Nice. LGTM.

**Comment by lioncash** [comment]
Can you squash your commits into one? Looks good after that.

**Comment by lioncash** [comment]
@dolphin-emu-bot rebuild

**Comment by phire** [comment]
LGTM

**Pull request merged**